src/components/composer/standalone_composer/standalone_composer_store.ts

```diff
@@ -0,0 +1,33 @@
+import { Get } from "../../../store_engine";
+import { CellPosition } from "../../../types";
+import { AbstractComposerStore } from "../composer/composer_store";
+
+export interface StandaloneComposerArgs {
+  onConfirm: (content: string) => void;
```

Comment:
Do you need the onConfirm here ?

Reply:
The confirm is called when a stopEdition is fired yeah